ksqldb-functional-tests/src/main/java/io/confluent/ksql/test/tools/TestExecutor.java

```diff
@@ -198,9 +198,9 @@ public void buildAndExecuteQuery(
           );
         }
 
-        topologyTestDriverContainer.getTopologyTestDriver()
-            .producedTopicNames()
-            .forEach(topicInfoCache::get);
+//        topologyTestDriverContainer.getTopologyTestDriver()
+//            .producedTopicNames()
+//            .forEach(topicInfoCache::get);
```

Comment:
FK-join creates a subscription and response topic, that the framework cannot handle yet... removed this now to be able to run tests

Reply:
Are we planning to add support for this in this PR or a follow-up PR? If a follow-up PR, we should gate this with the feature flag instead (if possible) as this is an important piece of testing.

Reply:
Not sure how complex it will be? Personally happy to do it both ways...
 - either fix in this PR
 - or keep QTT test disabled to merge this PR and do a fix in a follow up PR
 
 What do you think is better?

Reply:
I'm fine either way. If this PR is close to being merged, let's merge first to make it easier to parallelize subsequent work. We should address the open questions around the potential addition of extra state stores, and back out the changes to TableSelect but I think all the other comments/discussions can be addressed in follow-ups.